389.find-the-difference.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/find-the-difference/
+ */
+
+/**
+ * @param {string} s
+ * @param {string} t
+ * @return {character}
+ */
+var findTheDifference = function(s, t) {
+    let sArr = s.split("").sort();
+    let tArr = t.split("").sort();
```

Comment:
```suggestion
    const sArr = s.split("").sort();
    const tArr = t.split("").sort();
```
It is better to use const here